{
  "metadata": {
    "timestamp": "2025-02-26T06:16:05Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2020-24372",
        "description": "LuaJIT through 2.1.0-beta3 has an out-of-bounds read in lj_err_run in lj_err.c.",
        "metadata": {
          "published_date": "2020-08-17T17:15:13.987",
          "last_modified_date": "2024-11-21T05:14:41.440",
          "confidence_level": "low",
          "severity": "high"
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "none",
          "automatable": true
        },
        "counts": {
          "public_exploit_count": 0
        },
        "epss": {
          "score": 0.00124,
          "percentile": 0.48084
        },
        "metrics": [
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
            "attack_vector": "NETWORK",
            "base_score": 7.5
          },
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "2.0",
            "vector_string": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
            "attack_vector": "NETWORK",
            "base_score": 5
          }
        ],
        "timeline": {
          "references": [],
          "repositories": []
        }
      }
    }
  ]
}